kuba-kostrzewski/simpleToDoApp/src/test/java/pl/kostrzej/simpleToDoApp/app/MainMenuOptionsTest.java

```diff
@@ -0,0 +1,30 @@
+package pl.kostrzej.simpleToDoApp.app;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MainMenuOptionsTest {
+
+    @Test
+    public void chooseCorrectOption(){
+        assertEquals(MainMenuOptions.SHOW_ALL_TASKS,MainMenuOptions.returnIfCorrect(1));
+        assertEquals(MainMenuOptions.ADD_NEW_TASK,MainMenuOptions.returnIfCorrect(2));
+        assertEquals(MainMenuOptions.EXIT,MainMenuOptions.returnIfCorrect(3));
+    }
+
+    @Test(expected = InvalidOptionException.class)
+    public void choseTooHighOption(){
+        MainMenuOptions.returnIfCorrect(4);
```

Comment:
Uzależniłbym ten test od MainMenuOptions.values().length + 1. Dodając nowe opcję nie będziesz musiał poprawiać testów.

Reply:
Wydaje mi się, że zmiana wartości w teście powinna być świadoma i niezależna od klasy, która jest testowana. W przypadku gdy np. niechcący usunę jedną opcję to testy przejdą pozytywnie, a tak na prawdę nie powinny bo opcja, która jest używana w programie została usunięta.